dashboard/app/helpers/levels_helper.rb

```diff
@@ -479,10 +479,10 @@ def blockly_options
       set_unless_nil(level_options, 'failureMessageOverride', l.localized_failure_message_override)
 
       # Unintuitively, it is completely possible for a Blockly level to use
-      # Droplet, so we need to confirm the editory style before assuming that
+      # Droplet, so we need to confirm the editor style before assuming that
       # these fields contain Blockly xml.
       unless l.uses_droplet?
-        set_unless_nil(level_options, 'toolbox', l.localized_toolbox_blocks)
+        set_unless_nil(level_options, 'toolbox', Blockly.localize_toolbox_blocks(level_options['toolbox']))
```

Comment:
Would it make more sense to move this localize call somewhere here, where this value is initially set? https://github.com/code-dot-org/code-dot-org/blob/b92ebcc7d9e751c15f4d86a87c03ccf711c65daa/dashboard/app/models/levels/blockly.rb#L289-L292

Alternatively, this chunk of code could be extracted into something like a `localize_level_options` method if you're trying to avoid sprinkling localization code throughout a bunch of different places.